Documentation/advanced-configuration.md

```diff
@@ -26,6 +27,78 @@ The Kubernetes based examples assume Rook OSD pods are in the `rook-ceph` namesp
 If you run them in a different namespace, modify `kubectl -n rook-ceph [...]` to fit
 your situation.
 
+## Use custom Ceph user and secret for mounting
+
+> **NOTE** For extensive info about creating Ceph users, consult the Ceph documentation: http://docs.ceph.com/docs/mimic/rados/operations/user-management/#add-a-user.
+> Using a custom Ceph user and secret can be done for filesystem and block storage.
+
+Create a custom user in Ceph with access to just read write access in `/bar` directory on CephFS:
```

Comment:
what about this? 
`Create a custom user in Ceph with read-write access in the /bar directory on CephFS`